lib/Backends/OpenCL/OpenCLTensorLayout.cpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "OpenCLTensorLayout.h"
+#include "glow/Optimizer/GraphOptimizer/CompilationContext.h"
+
+using namespace glow;
+
+/// Definitions of different tensor layouts.
+static TensorDimensionDescription dimsNHWC[] = {
```

Comment:
It is better to prefix the names of these static arrays with something, so that they do not look exactly the same as names in other CPP files. This would help when debugging. Otherwise a debugger will see to static variables with the same name and it would be almost impossible to refer to each of them.